docs/plugins/orders.rst

```diff
@@ -17,10 +17,22 @@ Usage
     manager orders. It will not clear the orders that already exist.
 ``orders clear``
     Deletes all manager orders in the current embark.
+``orders reset``
```

Comment:
I'd call this "recheck", not "reset". "reset" is too similar to "clear" in semantics.

Reply:
and it might also be useful to differentiate between UI states -- `orders recheck` when viewing the details for a particular order could just reset that order, whereas it could reset all orders if called at any other time. alternately, the syntax could be `orders recheck [this]` where if `this` is specified, it will just reset the currently selected order, and all orders otherwise.

Reply:
Changed 'reset' to 'recheck'. 
We already have keybing for a single order on order conditions screen. Do you mean to rewrite it?

Reply:
yes, I think it makes sense to merge that into the `orders` plugin.

Reply:
Added `this` keyword to recheck only selected order. Moved single order invalidation to the orders plugin.